src/plugin/iframe_root/modules/components/signupForm.js

```diff
@@ -1156,23 +1181,26 @@ define([
                         padding: '2px',
                     },
                     dataBind: {
-                        class: 'selectedReferralSources.validationFieldBorder'
-                    }
+                        class: 'selectedReferralSources.validationFieldBorder',
+                    },
```

Comment:
Trailing commas, yes,?
I've avoided them, even though I know the benefits for diffs and copy-paste, due to the edge cases (e.g. copy/pasting to/from JSON), and also the ragged, unfinished vibe they bring.
But if we do use them, then I'd like to encode this into linting/formatting rules which enforce it, since I will never remember!

Reply:
I'm ambivalent. Up to you all. But we should try to be consistent.